src/kit/accessory.factory.js

```diff
@@ -5,27 +5,32 @@ function accessories(log, accessoryMaker, loginPromise) {
   return (homeBridgeCallback) => {
 
     loginPromise.then(authData => {
-      const { session } = authData;
+      session = authData.session
```

Comment:
`const { session } = authData` achieve the samething.

Reply:
I actually had to define `session` as a variable [here](https://github.com/jfdoube/homebridge-sinope/pull/6/files#diff-5bf60c416b72e205d0c9a321a8a07b67R12) so I could re-use it when [logging out](https://github.com/jfdoube/homebridge-sinope/pull/6/files#diff-5bf60c416b72e205d0c9a321a8a07b67R80), but I'm sure there's a better way of doing it!

Reply:
I think you get what you need with
```
const { session } = authData
```